src/cpp/src/visual_language/qwen2_5_vl/classes.cpp

```diff
@@ -152,13 +163,20 @@ ov::Tensor InputsEmbedderQwen2_5_VL::run_image_embeddings_merger(
     );
     size_t hidden_states_size = attention_mask.get_shape().at(1);
     ov::Tensor window_attention_mask = qwen2_5_vl_utils::get_window_attention_mask(hidden_states_size, cu_window_seqlens);
+    ov::Tensor t_cu_window_seqlens = qwen2_5_vl_utils::get_cu_window_seqlens(cu_window_seqlens);
 
     CircularBufferQueueElementGuard<ov::InferRequest> infer_request_guard(this->m_ireq_queue_vision_embeddings_merger.get());
     ov::InferRequest& vision_embeddings_merger = infer_request_guard.get();
     vision_embeddings_merger.set_tensor("hidden_states", concatenated_embeds);
-    vision_embeddings_merger.set_tensor("attention_mask", attention_mask);
+    if (m_with_cu_seqlens_input) {
+        vision_embeddings_merger.set_tensor("cu_seq_lens", cu_seq_lens);
+        vision_embeddings_merger.set_tensor("cu_window_seqlens", t_cu_window_seqlens);
+    }
+    else {
+        vision_embeddings_merger.set_tensor("attention_mask", attention_mask);
+        vision_embeddings_merger.set_tensor("window_attention_mask", window_attention_mask);
```

Comment:
Move computation of corresponding Tensors under `if` or `else` to avoid computing both of the paths

Reply:
done